pkgs/development/libraries/libaom/default.nix

```
@@ -45,7 +45,7 @@ stdenv.mkDerivation rec {
   ] ++ lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [
     # CPU detection isn't supported on Darwin and breaks the aarch64-darwin build:
     "-DCONFIG_RUNTIME_CPU_DETECT=0"
-  ] ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
+  ] ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform && !stdenv.hostPlatform.isStatic) [
```

I think the real issue here is that libaom uses yasm/nasm for x86 rather than GNU as
```suggestion
  ] ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform && !stdenv.hostPlatform.isx86) [
```

---

That makes sense, thanks!